__integrations__/__tests__/models/lollipopkeys.test.ts

```diff
@@ -0,0 +1,166 @@
+import { createContext } from "../../utils/cosmos_utils";
+import {
+  LolliPOPKeysModel,
+  LOLLIPOPKEYS_MODEL_PK_FIELD
+} from "../../../model/lollipop_keys";
+import {
+  aLolliPopPubKeys,
+  aPendingLolliPopPubKeys
+} from "../../../__mocks__/lollipopkeysMock";
+
+import { PubKeyStatusEnum } from "../../../generated/definitions/internal/PubKeyStatus";
+
+const context = createContext(LOLLIPOPKEYS_MODEL_PK_FIELD);
```

Comment:
Add the option to create the AzureStorage.

Reply:
Done in [d4b4041](https://github.com/pagopa/io-functions-lollipop/pull/19/commits/d4b40412b91b5e186f0499216eb512fe954c394c)